core/domain/question_services.py

```diff
@@ -117,6 +119,23 @@ def get_question_by_id(question_id):
         return None
 
 
+def get_questions_by_ids(question_ids):
+    """Returns a list of domain objects representing questions.
+
+    Args:
+        question_ids: list(str). List of question ids.
+
+    Returns:
+        list or None. A list of domain objects representing question with the
```

Comment:
When exactly is None returned?

Reply:
There is possibility that question_ids is empty or it can have a list of deleted question_ids.

Reply:
OK, please explain in the docstring. That said, shouldn't we return an empty list in that case instead?

Btw, the type should be list(Question). (or whatever the domain object in the list is called).

Reply:
updated.